src/pages/settings/Profile/PronounsPage.js

```diff
@@ -106,51 +66,76 @@ class PronounsPage extends Component {
             })
             .sortBy((pronoun) => pronoun.text.toLowerCase())
             .value();
-    }
+
+        setPronounsList(pronouns);
+    }, [props]);
```

Comment:
@aimane-chnaif I have removed the de-structuring as per your previous comments & guidelines but just highlighting the use of [props] in the deps array & this warning

<img width="832" alt="Screenshot 2023-06-09 at 4 03 26 AM" src="https://github.com/Expensify/App/assets/38547776/bfa316db-1a21-47a4-8255-ddd118f30b5b">

Reply:
@aimane-chnaif this lint error is what I was referring to earlier, please advice.

![image](https://github.com/Expensify/App/assets/38547776/44a9018d-3048-4391-9d49-f5dfb649fb20)